src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -118,35 +235,35 @@
   </div>
   <div class="section-body">
     <table>
-      {% if ccj.signerName is empty %}
+      {% if ccj.claim.claim.statementOfTruth.signerName is empty %}
         <tr>
           <td>
             Name:
           </td>
-          <td>{{ ccj.claimantName}}</td>
+          <td>{{ claimant.name }}</td>
         </tr>
       {% endif %}
 
-      {% if ccj.signerName is not empty %}
+      {% if ccj.claim.claim.statementOfTruth.signerName is not empty %}
         <tr>
           <td>
             Organisation:
           </td>
-          <td>{{ ccj.claimantName }}</td>
+          <td>{{ claimant.name }}</td>
         </tr>
 
         <tr>
           <td>
             Name:
           </td>
-          <td>{{ ccj.signerName }}</td>
+          <td>{{ ccj.claim.claim.statementOfTruth.signerName }}</td>
```

Comment:
Shouldn't we take the signer name and signer role from CCJ not a claim?

Reply:
correct, oops